nikolai4D/frontEndFramework
# Uc 98 add project info modal and create content to it to project view

UC-98-add-project-info-modal-and-create-content-to-it-to-project-view


## Description & motivation

Creating a modal for project view.
No other functionality added execept for adding eventlisteners. 



## How to test

Switch to branch #UC-98-add-project-info-modal-and-create-content-to-it-to-project-view in UrbanCloud.
Build and start project. 
Click on a card to see modal. 



## New tickets to be added

Functionalities for this ticket.



---
